package.json

```diff
@@ -42,7 +42,7 @@
     "loopback": "^2.22.0",
     "loopback-boot": "^2.6.5",
     "loopback-component-explorer": "^2.1.0",
-    "loopback-datasource-juggler": "^2.39.0",
+    "loopback-datasource-juggler": "^3.0.0",
```

Comment:
@bajtos : Yes, I noticed that and corrected it on Local. But facing failures, trying to resolve them and didn't push yet.